### ISSUE TITLE
Research: How application/job information is split between TTS and 18F join sites

### ISSUE THREAD

**Issue by awfrancisco** [18F/18f.gsa.gov#2817]
Information about open positions, how to apply, pay, and benefits is currently both on join.18f.gov and join.tts.gsa.gov. The information on join.18f.gov is designed to encourage people to apply, while join.tts.gsa.gov has more comprehensive information that includes info on the process and how to submit an application. The TTS site is also the official home of all job descriptions. The 18F site links to TTS for individual job openings. 

The research question should be directed at learning more about how potential candidates find out about open positions, how they decide to apply, and their experience with these two places for information. This research will be used to decided what join content remains on the 18F site or if all content should be removed and only link to the TTS site. 

## This issue is done when:
- [x] Create [research plan](https://docs.google.com/document/d/1KLLxPFSBGjhWPm7LC9c8OR3r2fn7rN938TuZBUcqvd8/edit?usp=sharing)
- [x] Recruit participants
- [x] Conduct research
- [x] Synthesize findings

**Comment by Dahianna** [comment]
[Findings and recommendations](https://docs.google.com/document/d/1OLBRdNj-qMEsGxY7DOc2-zYNPFGb8ZE4RWriyxTfT1U/edit?usp=sharing)

**Comment by Dahianna** [comment]
**TLDR**
Recommendations
- Add teleworking as a benefit for all employees, not just remote positions.
- Try to reduce the amount of text by using graphics or linking out to resources.
- We can include a short timeline similar to the one in the TTS Join site.
- Some candidates looked through the site and found the project pages. We could linked to the project pages from the Join page to make it easier for candidates to find this information.
- Add a small section about the 18F chapters We could take some content from the handbook or we can linked out to it, so we don't need to worry about updating. 
- Bring back the testimonials to the join page.
- Include a link to the TTS page from the 18F Join page for applicants interested in learning about other openings outside 18F. For example:
 "18F is not hiring right now, but checkout other positions with the Technology Transformation Services (TTS)"

**Comment by eddietejeda** [comment]
@awfrancisco will write new issue outline an issue describing what language can be used on the 18F Join page.
